_config.yml

```diff
@@ -24,36 +24,47 @@
 # twitter_username: jekyllrb
 # github_username:  jekyll
 
-contributions_to_date: $30,948
+contributions_to_date: $36,212
 
 current_focus:
-  title: Environmental Justice
+  title: Freedom to Vote
   # Note: have to cap off the body with "…" manually:
-  body: "With the United States announcing its withdrawal from the Paris Climate Agreement, our community has decided to again focus on environmental justice. Over 50% of our members voted for this focus, demonstrating how critical we feel this issue is in the fight for a more equitable and just world."
-  url: https://blog.vocalminority.us/vocal-minority-funds-climate-education-and-action-41b473dc2e42
+  body: "We're focusing this month on freedom—one of the bedrocks of our country. Specifically, our community has decided to focus on our freedom to vote. We believe that all Americans have the right to express their beliefs through the power of our voting system. Given the nationwide discussion regarding voter suppression, we've decided to fight back this month."
```

Comment:
Can we get it closer to something like the original prompt:

> The Presidential Advisory Commission on Election Integrity has requested highly-sensitive personal information on every U.S. voter—from social security numbers to dates of birth—in the name of preventing voter fraud. Why is the federal government marshaling considerable taxpayer resources to collect data it shouldn’t have, to reach a conclusion it cannot possibly reach, to prove something we already know isn’t true? To counter this careless act, we'll support an organization fighting to preserve the power of your vote.

It dives into the meat of the issue a little faster.